### DISCUSSION TITLE
Backup download failed

### DISCUSSION THREAD

**Discussion by Sagnasty** [rern/rAudio#1305] [Fixed Issues]
Running rAudio 20230310

Reproduced on  Raspberry Pi 3B Rev 1.2 and Raspberry Pi 3B+ Rev 1.3

![image](https://github.com/rern/rAudio/assets/76575081/7cfe6c92-b62f-4705-86c4-8fbe6a610760)

3 x Raspberry Pi 3B+ Rev 1.3 > HiFiBerry DAC+ Pro
1 x Raspberry Pi 3B Rev 1.2 > HiFiBerry DAC+ Pro
Synology DS718+

Jim

**Comment by klassikmann** [comment]
Same here (rAudio 20230522)

**Reply by Sagnasty** [reply]
Correction: I am running rAudio 20230522

**Reply by Lubos-V** [reply]
Have been observing this for quite some time now, but always forgot to report it. So, I can confirm this is indeed happening... and has been for at least several last weeks.

**Comment by klassikmann** [comment]
rAudio 20230523: Fixed
